cmd/ctr/commands/plugins/plugins.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/containerd/containerd/cmd/ctr/commands"
 	"github.com/containerd/containerd/platforms"
 	pluginutils "github.com/containerd/containerd/plugin"
-	v1 "github.com/opencontainers/image-spec/specs-go/v1"
+	ocispec "github.com/opencontainers/image-spec/specs-go/v1"
```

Comment:
Maybe:
```suggestion
	imagespec "github.com/opencontainers/image-spec/specs-go/v1"
```

Because we have runtime-spec and distribution-spec

Reply:
Same suggestion: https://github.com/containerd/containerd/pull/8566#discussion_r1204864160